dapp/src/components/Layout.tsx

```diff
@@ -1,27 +1,43 @@
 import React from "react"
 import { Outlet } from "react-router-dom"
 import { Flex } from "@chakra-ui/react"
-import { useMobileMode } from "#/hooks"
+import { useIsEmbed, useMobileMode } from "#/hooks"
 import DocsDrawer from "./DocsDrawer"
 import Header from "./Header"
 import ModalRoot from "./ModalRoot"
 import Sidebar from "./Sidebar"
+import MobileModeBanner from "./MobileModeBanner"
+
+const PADDING = "2.5rem" // 40px
+const PAGE_MAX_WIDTH = {
```

Comment:
Both values are exact the same. Why do we differentiate it?

Reply:
The value for the standalone dApp will be updated soon. More info [here](https://discord.com/channels/1079490158639976609/1295738070167191573/1296402533623009301)